cmd/ocsp-updater/main_test.go

```diff
@@ -496,6 +496,10 @@ func TestMissingReceiptsTick(t *testing.T) {
 }
 
 func TestMissingOnlyReceiptsTick(t *testing.T) {
+	// Enable the ResubmitMissingSCTsOnly feature flag for this test run
```

Comment:
Why is this moved? And slightly worryingly why doesn't it break anything... (or was it broken before?)